pkgs/applications/editors/texmacs/default.nix

```diff
@@ -26,21 +26,22 @@ stdenv.mkDerivation {
 
   src = fetchurl {
     url = "https://www.texmacs.org/Download/ftp/tmftp/source/TeXmacs-${version}-src.tar.gz";
-    sha256 = "12bp0f34izzqimz49lfpgf4lyz3h45s9xbmk8v6zsawdjki76alg";
+    sha256 = "Aq0cS47QqmFQHelxRjANeJlgXCXagnYRykpAq7wHqbQ=";
   };
 
-  cmakeFlags = [
-    # Texmacs' cmake build as of writing defaults to Qt5,
-    # but we haven't updated to that yet.
-    "-DTEXMACS_GUI=Qt4"
+  patches = [
+    (fetchpatch {
```

Comment:
Please comment what is the patch for, and when it is likely to be not needed.

Reply:
Good point, done!